src/Utils/StringUtils.ts

```diff
@@ -18,7 +18,7 @@ export class StringUtils {
     public static normalize(value: any, defaultValue?: any): any {
```

Comment:
could you change normalize, I think it's incorrect, for example `value = 0`

Reply:
And, this is string normalize, so at first you should check if the value is a string, if NOT, do nothing